Gaijinizer patch/Map122.txt

```diff
@@ -618,27 +641,31 @@ Panic. That is the measure.#MTLed
 おかげで、スライムみたいな液状の悪魔ですら
 問題なく捕獲できるんだよ。
 > CONTEXT: Map122/events/5/pages/1/290/Dialogue/enemy14
-In addition to that, there is also a function to
-shut off the flow of magic. Thanks to you, even a
-liquid demon like slime. You can capture without
-problems.#MTLed
+On top of that, it's also got the ability to cut
+off the flow of magic. Thanks to that, I can even
+capture liquid demons like slimes without much
+of a problem.#Ron
 > END STRING
```

Comment:
"On top of that," -> "Additionally," (no reason to use to "that's" to be repetitive.)